server/build/babel/preset.js

```diff
@@ -7,10 +7,21 @@ const productionPlugins = isProduction ? [
   require.resolve('babel-plugin-transform-react-remove-prop-types')
 ] : []
 
+const es2015Config = {}
+
+// Do not transform ES2015 modules if we are inside Next.js
+// That's because webpack2 knows how to handle it.
+// (And that's how it can do code splitting)
+//
+// But in other environements like Jest, we should transform it.
+if (process.env.INSIDE_NEXT) {
```

Comment:
What if you add a config option to disable module transpilation 🤔 So they are enabled by default when no options are given. And in our code (webpack.js) we pass the disabled option. Would that work? 🤔